automation_oca/models/automation_record_step.py

```diff
@@ -124,6 +143,7 @@ def run(self, trigger_activity=True):
             return self.browse()
         if (
             self.record_id.resource_ref is None
+            or not self.configuration_step_id
```

Comment:
@luisDIXMIT when the cron reaches this, it will not detect the configuration and the execution will be rejected.